packages/core/src/config/types.ts

```diff
@@ -64,9 +64,10 @@ export type ParsedVariable =
       [name: string]: ParsedVariable
     }
 
-export type RawActionInput = RawFunctionCallActionInput | RawCreate2ActionInput
```

Comment:
The `RawActionInput` types are no longer necessary, so I removed them. Previously, we used them as an intermediate data type between the collection and gas estimation phase

Reply:
Got it, good to know. I'll make a ticket for updating Mean Finance's integration (and anybody else's)